ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -30,23 +32,21 @@
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
     }
 
-    @Ignore("TODO fix, inpatient_ward doesn't exisit on the page, fix workflow")
     @Test
-    public void transferToWardServiceTest() throws Exception {
-
-        ClinicianFacingPatientDashboardPage patientDashboardPage = homePage.goToActiveVisitPatient();
-
-        patientDashboardPage.goToAdmitToInpatient().confirm(INPATIENT_WARD);
-        patientDashboardPage.waitForPage();
-
-        patientDashboardPage.goToTransferToWardServicePage().confirm(ISOLATION_WARD);
-        patientDashboardPage.waitForPage();
-
-        assertTrue(patientDashboardPage.containsText("Outpatient"));
+    @Category(BuildTests.class)
+    public void transferToWardServiceTest() {
```

Comment:
Refactored the `test logic implementation` to suit the right test workflow for `transferToWardService` test